src/ureadahead.c

```diff
@@ -113,7 +113,7 @@ static char *path_prefix_filter = NULL;
 /**
  * use_existing_trace:
  *
- * Set to TRUE if tracing events (tracing/events/fs/*) used to build the pack
```

Comment:
This is a silly warning, as it really wants to be "tracing/events/fs/*" as that is the list of events. I wonder if we can make the compiler not warn about that.

Reply:
According to a simple test of my own, this is:

```
warning: '/*' within block comment [-Wcomment]
```

clang warns by default; gcc requires `-Wall`. And we could disable with `-Wno-comment`. (Works on both gcc and clang for me.)

But I also see even my syntax highlighter complains.

It's just as accurate and readable to use the asterisk as not IMO, so I'd say the "fix" is fine?

Plus, Alan's other PR already changed this same line, so now there's merge conflicts :)